src/Riverline/MultiPartParser/StreamedPart.php

```diff
@@ -36,18 +36,27 @@ class StreamedPart
      */
     private $parts = array();
 
+    /**
+     * The length of the EOL character.
+     *
+     * @var int
+     */
+    private $EOLCharacterLength = 2;
+
     /**
      * StreamParser constructor.
      *
      * @param resource $stream
+     * @param int $EOLCharacterLength
      */
-    public function __construct($stream)
+    public function __construct($stream, $EOLCharacterLength = 2)
     {
         if (false === is_resource($stream)) {
             throw new \InvalidArgumentException('Input is not a stream');
         }
 
         $this->stream = $stream;
+        $this->EOLCharacterLength = $EOLCharacterLength;
```

Comment:
Can you add a type check here as I did with the stream ?

Reply:
Done.